homeassistant/components/android_ip_webcam/const.py

```diff
@@ -0,0 +1,189 @@
+"""Constants for the Android IP Webcam integration."""
+
+from datetime import timedelta
+from typing import Final
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.components.switch import SwitchEntityDescription
+from homeassistant.helpers.entity import EntityCategory
+
+DOMAIN: Final = "android_ip_webcam"
+DEFAULT_NAME: Final = "IP Webcam"
+DEFAULT_PORT: Final = 8080
+DEFAULT_TIMEOUT: Final = 10
+
+CONF_MOTION_SENSOR: Final = "motion_sensor"
+
+MOTION_ACTIVE: Final = "motion_active"
+SCAN_INTERVAL: Final = timedelta(seconds=10)
+
+
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
```

Comment:
Please move the entity descriptions to their respective platforms. The description is a platform detail.